devicetypes/smartthings/zigbee-range-extender.src/zigbee-range-extender.groovy

```diff
@@ -0,0 +1,61 @@
+/**
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Zigbee Range Extender", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.networking") {
+		capability "Health Check"
+
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0009, 0B05, 1000, FC7C", outClusters: "0019, 0020, 1000", model: "TRADFRI signal repeater", deviceJoinName: "TRÅDFRI Signal Repeater"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "status", type: "generic", width: 6, height: 4) {
+			tileAttribute("device.status", key: "PRIMARY_CONTROL") {
+				attributeState "online", label: 'online', icon: "st.motion.motion.active", backgroundColor: "#00A0DC"
+			}
+		}
+		main "status"
+		details(["status"])
+	}
+}
+
+def installed() {
+	runEvery5Minutes(ping)
```

Comment:
@varzac  is there a way to configure automatic reporting at the zigbee protocol level here instead of this?

Reply:
I believe you should just be able to do  `zigbee.configureReporting(BASIC_CLUSTER, ZCL_VERSION_ATTRIBUTE, DataType.ENUM8, 240, 300, null)` in the configure method.  However, I believe we've had trouble with some things like this in the past, so it probably depends on the device.  But I would definitely try it and see if it works as avoiding cloud schedules would be nice.

Reply:
Yeah, I imagine it might take a little trial and error to find a reportable attribute but agree it would be preferable than using the schedule

Reply:
I've tried couple of attributes from basic cluster, but none of them was reporting correctly. 
Device does responds to ping() though, so I'm not sure if trying to hit the jackpot with suitable attribute is worth the time.

Reply:
Well, if some time up front can save us platform load for the rest of time it's probably worth it.  The device says it supports the alarm cluster, so you could also try the following cluster, attribute, datatype:
0x0009, 0x0000, DataType.UINT16

Reply:
Actually I thought about deleting scheduling, and leaving taking care of device's status to Health Check capability. I've kept this so far, as this was also used in Z-Wave range extender, but now I think it's kinda overdoing.

Reply:
Anyway, reporting on this cluster doesn't work neither.